AdminWebsite/AdminWebsite/ClientApp/src/app/services/clients/api-client.ts

```diff
@@ -24,7 +24,7 @@ export class BHClient {
 
     constructor(@Inject(HttpClient) http: HttpClient, @Optional() @Inject(BH_API_BASE_URL) baseUrl?: string) {
         this.http = http;
-        this.baseUrl = baseUrl ? baseUrl : 'https://localhost:5400';
+        this.baseUrl = baseUrl !== undefined && baseUrl !== null ? baseUrl : 'https://localhost:5400';
```

Comment:
I don't think this adds anything - both undefined and null are falsey. The only difference will be that with the new code, if baseUrl is an empty string or some other falsey value, it'll get used - whereas previously those values would have been replaced with `'https://localhost:5400'`

Reply:
Ignore this, didn't realise it was autogenerated

Reply:
this file (api-client.ts) is auto generated